### ISSUE TITLE
update with github pages

### ISSUE THREAD

**Issue by PatrickJS** [jwcarroll/angular2-databinding-examples#1]
it would be great to have a link to the demo

**Comment by PatrickJS** [comment]
it would be great to also have react/angular1 examples

**Comment by jwcarroll** [comment]
@gdi2290 Yes, I have plans to include an ng1 example for comparisson, and I agree completely on the github pages part. I started to look at this yesterday, but got busy with other things.
